### PR TITLE
jooby: netty implementation

### PR DESCRIPTION
**Pull request by jknack**
- move from undertow to netty web server

### CODE REVIEW

**Review by waghanza — DISMISSED** [review]
LGTM but you know you can have two folders (jooby-undertow and jooby-netty) if you want.

And you can also do the same for kotlin

@jknack

**Comment by jknack** [comment]
@waghanza didn't know about multiple folder/server, thank you. For now let's keep just one and let me update the kotlin version

File: kotlin/kooby/config.yaml [hunk 1]
```diff
@@ -1,3 +1,11 @@
 framework:
   website: jooby.io
   version: 3.0
+
+  command: >
```
Comment (jknack) [inline kotlin/kooby/config.yaml:5]: @waghanza added these lines for kotlin implementation

**Review by waghanza — APPROVED** [review]

**Comment by waghanza** [comment]
is there a reason that jooby is named `kooby` in `kotlin` @jknack ?

**Comment by waghanza** [comment]
I think it could be a good thing to rename it, but if you agree I'll do it (in another PR when I craft another UI)

Ofc if you are ok with that @jknack

**Comment by jknack** [comment]
@waghanza sure. Thank you.

**Review by waghanza — APPROVED** [review]

**Pull request merged**